src/routes/product/[id([0-9]+)]/edit.svelte

```diff
@@ -295,7 +295,7 @@
   <link rel="stylesheet" href="css/page-components/footer.css" />
 </svelte:head>
 
-<Layout>
+<Layout user={account}>
```

Comment:
So if you moved away from the `$session.user`, I suppose there's no longer a reason to set it on every page.

Reply:
To be precise, I'm talking about lines like https://github.com/VanishMax/innopoints-frontend/blob/a994c3521f94b81842a4d2e8c77df155fd5aa8be/src/routes/profile.svelte#L48 which I used to write before

Reply:
Should I just remove this or all of them from everywhere? Note that there might be some components that still make use of it without accepting it as a prop yet, but that should also be fixed and isn't too difficult

Reply:
Actually, nvm. It took 2 minutes to remove them all